test-utils/runtime/Cargo.toml

```
@@ -97,3 +97,5 @@ std = [
 	"sp-transaction-pool/std",
 	"trie-db/std",
 ]
+# Special feature to disable logging
+disable-logging = [ "sp-api/disable-logging", "log/max_level_error" ]
```

why is it "log/max_level_error" here instead of "log/max_level_off" don't we want to disable logging ?

---

That was me trying 3 hours to get this running, until I realized why it doesn't work.. Actually we need to remove this. Good find! :) 
```suggestion
disable-logging = [ "sp-api/disable-logging" ]
```